### ISSUE TITLE
The self built training set cannot converge

### ISSUE THREAD

**Issue by HLegend** [TIGER-AI-Lab/ConsistI2V#32]
I use this model to train my self built training set, but the loss curve is quite strange and cannot converge. Can you give me some useful suggestions

![Image](https://github.com/user-attachments/assets/a1d317d0-75fc-4f5b-ab83-bf2a87666aad)

**Comment by wren93** [comment]
Hello, I believe the loss for training these stable diffusion models won't represent much of the actual model generation capability. The timestamps are randomly sampled during training, meaning that for some timestamps (e.g. smaller t, for which the noisy latent already contains a large amount of information) you can easily get a low loss value as denosing from such noisy latents will be very easy. So your loss curve may not have a clear convergence trend but that won't affect your model performance improving throughout the training.
